flang/include/flang/Optimizer/Dialect/FIROps.td

```diff
@@ -368,15 +368,15 @@ def fir_PrefetchOp : fir_Op<"prefetch", []> {
 
   /// `memref' is the address to be prefetched
   /// `rw'          : rw specifier >
-  ///                  read is 0, write is 1
+  ///                  read is 0 (default), write is 1
   /// `localityHint': temporal locality specifier >
   ///                  value ranging from 0 - no locality to 3 - extremely local
   /// `cacheType'   : cache type specifier >
-  ///                  instruction cache is 0, data cache is 1
-  let arguments = (ins AnyReferenceLike:$memref,
-      ConfinedAttr<I32Attr, [IntMinValue<0>, IntMaxValue<1>]>:$rw,
+  ///                  instruction cache is 0 (default), data cache is 1
```

Comment:
Wouldn't `data cache` be the better default? Because in the way this is used in flang I am not sure how you would even trigger an instruction cache prefetch.